lib/tether/ops_darwin.go

```diff
@@ -96,24 +95,58 @@ func (t *BaseOperations) Cleanup() error {
 	return nil
 }
 
-// Need to put this here because Windows does not
-// support SysProcAttr.Credential
-func getUserSysProcAttr(uname string) *syscall.SysProcAttr {
```

Comment:
I don't know, the original code duplicate the logic in ops_darwin, so I thought they are necessary.
But all other functions in ops_darwin is not implemented, so I guess we still do not tether darwin fully functional, so I'll remove the impl. @hickeng let me know, if you have any concern.